torch/_fullgraph.py

```diff
@@ -0,0 +1,52 @@
+import dataclasses
```

Comment:
Why putting this at the very top level torch namespace?

Reply:
No particular reason. I can move it to another place.